### PR TITLE
[non-forked] Source Stripe: New Stream - Early Fraud Warnings

### LINKED ISSUE #16962: Source Stripe: new stream Early Fraud Warning
## Tell us about the problem you're trying to solve
[Early Fraud Warning](https://stripe.com/docs/api/radar/early_fraud_warnings)
An early fraud warning indicates that the card issuer has notified us that a charge may be fraudulent..

[Permission](https://stripe.com/docs/security/permissions)

## Describe the solution you’d like
Research pros and cons for implementing this stream.
_Research should contain: what stream should be implemented, permissions and scopes (if needed), paid plan (if needed)._

### PR DESCRIPTION
**Pull request by natalyjazzviolin**
## What
Non-forked branch of https://github.com/airbytehq/airbyte/pull/22859 to run the build.  Closes https://github.com/airbytehq/airbyte/issues/16962

## Recommended reading order
1. `x.java`
2. `y.python`

## 🚨 User Impact 🚨
Are there any breaking changes? What is the end result perceived by the user? If yes, please merge this PR with the 🚨🚨 emoji so changelog authors can further highlight this if needed.

## Pre-merge Checklist
Expand the relevant checklist and delete the others.

<details><summary><strong>New Connector</strong></summary>

### Community member or Airbyter

- [ ] **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret`
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [ ] `docs/integrations/README.md`
    - [ ] `airbyte-integrations/builds.md`
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)
- [ ] After the connector is published, connector added to connector index as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
- [ ] Seed specs have been re-generated by building the platform and committing the changes to the seed spec files, as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)

</details>

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- [ ] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret`
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [ ] New Connector version released on Dockerhub and connector version bumped by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)

</details>

<details><summary><strong>Connector Generator</strong></summary>

- [ ] Issue acceptance criteria met
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)
- [ ] If adding a new generator, add it to the [list of scaffold modules being tested](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/generator/build.gradle#L41)
- [ ] The generator test modules (all connectors with `-scaffold` in their name) have been updated with the latest scaffold by running `./gradlew :airbyte-integrations:connector-templates:generator:testScaffoldTemplates` then checking in your changes
- [ ] Documentation which references the generator is updated as needed

</details>

### CODE REVIEW

**Comment by evantahler** [comment]
I updated the docker image and added a changelog entry

**Review by evantahler — APPROVED** [review]

File: airbyte-integrations/connectors/source-stripe/acceptance-test-config.yml [hunk 1]
```diff
@@ -16,46 +16,132 @@ acceptance_tests:
...
-            - name: network_token
-              bypass_reason: "Randomly added to the record"
-          payment_intents:
-            - name: receipt_url
-              bypass_reason: "URL changes upon each request for privacy/security"
-            - name: charges/data/*/receipt_url
-              bypass_reason: "URL changes upon each request for privacy/security"
-            - name: network_token
-              bypass_reason: "Randomly added to the record"
+    - config_path: "secrets/config.json"
```
Comment (evantahler) [inline airbyte-integrations/connectors/source-stripe/acceptance-test-config.yml:19]: @alafanechere I'm working on testing this community PR... and I can't get the YAML right.  Help!

**Comment by evantahler** [comment]
@roman-yermilov-gl can you please help us out here?

File: airbyte-integrations/connectors/source-stripe/source_stripe/schemas/early_fraud_warnings_with_id.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+			"$id": "#root/object", 
+			"title": "Object", 
+			"type":  ["null", "string"],
+			"default": "",
+			"pattern": "^.*$"
+		},
+		"actionable": {
+			"$id": "#root/actionable", 
+			"title": "Actionable", 
+			"type":  ["null", "string", "boolean"],
```
Comment (roman-yermilov-gl) [inline airbyte-integrations/connectors/source-stripe/source_stripe/schemas/early_fraud_warnings_with_id.json:25]: It looks like this type declaration is strange. So what is the exact type of this field (string or boolean)?

**Review by roman-yermilov-gl — CHANGES_REQUESTED** [review]

**Comment by evantahler** [comment]
Thank you for the review @roman-yermilov-gl!
@btkcodedev - can you make the updates and re-submit your PR?

**Comment by btkcodedev** [comment]
@evantahler Thanks for the comments, Resolved them and resubmitted at https://github.com/airbytehq/airbyte/pull/22859

**Comment by btkcodedev** [comment]
@evantahler @sajarin Thanks for the comments, Resolved them and resubmitted them at https://github.com/airbytehq/airbyte/pull/22859

**Pull request closed without merging**